modules/identity/pages/single-sign-on-with-oidc.adoc

```diff
@@ -15,7 +15,7 @@ OpenID Connect configuration is at tenant level. Each tenant can use a different
 
 [NOTE]
 ====
-Bonita uses http://www.keycloak.org/[Keycloak] OIDC Service client adapter. It has been extended to support other OIDC providers than Keycloak.  
+Bonita uses http://www.keycloak.org/[Keycloak] OIDC Service client adapter. It has been extended to support other OIDC providers than Keycloak.
```

Comment:
⚠️ autoformated by my IDE. If a break line was intended here, I can add a `+` symbol to enforce it

Reply:
looks better with a line break